include/sfx.h

```diff
@@ -627,6 +627,8 @@
 #define NA_SE_EV_YOBI24 0x2917
 #define NA_SE_EV_YOBI25 0x2918
 //------------ENEMY------------
+#define NA_SE_EN_FLOORMASTER_SLIDING 0x3034
+#define NA_SE_EN_FLOORMASTER_SM_STICK 0x3136
```

Comment:
I can't imagine this will be used anywhere else.  I can either hardcode the values back in the c file, create different macros for the playback bit being set, or however else you'd like to go about it.

Reply:
My point is all these sfx defines should either have the bit set, or none of them. I'm not sure changing them arbitrarily when they are only used one way or the other is a good idea.
And since we don't know how the value is processed in the audio code, we can't really design a proper pattern yet, so that'll have to wait a bit.

I'm mostly mentioning to make sure we're on the same page in that this is a temporary workaround, but it's fine for now and we can deal with it later.